platform/src/test/kotlin/org/stellar/anchor/platform/service/TransactionServiceTest.kt

```diff
@@ -3,8 +3,11 @@ package org.stellar.anchor.platform.service
 import io.mockk.*
 import io.mockk.impl.annotations.MockK
 import java.time.Instant
-import org.junit.jupiter.api.*
+import org.junit.jupiter.api.AfterEach
 import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
```

Comment:
Can these be removed?

Reply:
I am not sure if I mis-understand. But the import is required by the assertion tests used by [line#62](https://github.com/stellar/java-stellar-anchor-sdk/blob/hotfix/1.2.1-fix-refunds-gson-serialization/platform/src/test/kotlin/org/stellar/anchor/platform/service/TransactionServiceTest.kt#L63) and statements similar to that.

I will merge first. If it is still an issue, I will create another PR.